.bazelrc

```diff
@@ -21,6 +21,10 @@ build --java_runtime_version=remotejdk_11
 # https://github.com/GoogleContainerTools/rules_distroless/actions/runs/7118944984/job/19382981899?pr=9#step:8:51
 common:linux --sandbox_tmpfs_path=/tmp
 
+
+# Allow external dependencies to be retried. debian snapshot is unreliable and needs retries.
+common --experimental_repository_downloader_retries=200
```

Comment:
200 is a high number! would we actually want a build to succeed after spending the time to make 199 failed downloads?

also, I think this flag probably needs to go into bazel-lib presets, could you make a PR there too?

Reply:
Hah yeah that's pretty high, perhaps we just suggest people use the cloudflare snapshot mirror? and keep this number a bit lower. Distroless will probably be one of the most heavy consumers of this, people building one off will probably only need to download a few packages

Reply:
Upps, this is leftover from an experiment. Will lower it.